VERSION

```diff
@@ -1 +1 @@
-2.0.0-rc.1
+2.0.0
```

Comment:
This will collide with #2212

Reply:
An artefact of the PR still being against `minerva`. I changed the base to `master`.